cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,87 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
+  url = '/settings';
+
+  editUsername (username) {
+    cy.getByDataCy('usernameSettings')
+      .type('{selectAll}' + `${username}`);
+  };
+
+  editBio (bio) {
+    cy.getByDataCy('bioSettings')
+      .type('{selectAll}' + `${bio}`);
+  };
+
+  editEmail (email) {
+    cy.getByDataCy('emailSettings')
+      .type('{selectAll}' + `${email}`);
+  };
+
+  editPassword (password) {
+    cy.getByDataCy('passwordSettings')
+      .type('{selectAll}' + `${password}`);
+  };
+
+  submitSettings () {
+    cy.getByDataCy('submitSettings')
+      .click();
+    cy.wait(3000);
```

Comment:
remove wait. You almost never need to wait for an arbitrary period of time. There are always better ways to express this in Cypress.
https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
thanks! removed waits and committed